"use strict";(self.webpackChunkscvi_tools_site=self.webpackChunkscvi_tools_site||[]).push([[477],{10:function(e){e.exports=JSON.parse('{"blogPosts":[{"id":"v090","metadata":{"permalink":"/blog/v090","editUrl":"https://github.com/YosefLab/scvi-tools-site/blog/blog/2021-3-03-scvi-tools-v090.md","source":"@site/blog/2021-3-03-scvi-tools-v090.md","title":"scvi-tools 0.9.0 release","description":"Today we officially released scvi-tools version 0.9.0 (changelog). This release marks the culmination of five months of work on the backend of the codebase, which came after three months of work on the frontend.","date":"2021-03-03T00:00:00.000Z","formattedDate":"March 3, 2021","tags":[{"label":"scvi-tools","permalink":"/blog/tags/scvi-tools"},{"label":"release","permalink":"/blog/tags/release"}],"readingTime":6.11,"truncated":true,"authors":[{"name":"Adam Gayoso, Romain Lopez, Galen Xing, Nir Yosef"}],"nextItem":{"title":"Hyperparameter search for scVI","permalink":"/blog/autotune"}}},{"id":"autotune","metadata":{"permalink":"/blog/autotune","editUrl":"https://github.com/YosefLab/scvi-tools-site/blog/blog/2019-7-5-Hyperoptimization.md","source":"@site/blog/2019-7-5-Hyperoptimization.md","title":"Hyperparameter search for scVI","description":"A study of the effects of hyperparameter optimization on scVI models.","date":"2019-07-05T00:00:00.000Z","formattedDate":"July 5, 2019","tags":[],"readingTime":19.865,"truncated":true,"authors":[{"name":"Gabriel Misrachi, Jeffrey Regier, Romain Lopez, Nir Yosef"}],"prevItem":{"title":"scvi-tools 0.9.0 release","permalink":"/blog/v090"},"nextItem":{"title":"Should we zero-inflate scVI?","permalink":"/blog/zero-inflation"}}},{"id":"zero-inflation","metadata":{"permalink":"/blog/zero-inflation","editUrl":"https://github.com/YosefLab/scvi-tools-site/blog/blog/2019-6-27-ZeroInflation.md","source":"@site/blog/2019-6-27-ZeroInflation.md","title":"Should we zero-inflate scVI?","description":"Droplet- based single-cell RNA sequencing (scRNA-seq) datasets typically contain at least 90% zero entries. How can we best model these zeros? Recent work focused on modeling zeros with a mixture of count distributions. The first component is meant to reflect whether such an entry can be explained solely by the limited amount of sampling (on average ~5% or less of the molecules in the cell). The second component is generally used to reflect \\"surprising\\" zeros caused by measurement bias, transient transcriptional noise (e.g., \\"bursty\\" gene with a short mRNA half life), or true longer-term heterogeneity that can not be captured by a similified (low dimensional) representation of the data. Among others, zero-inflated distributions (i.e., zero-inflated negative binomial) have been widely adopted to model gene expression levels (1, 2).","date":"2019-06-25T00:00:00.000Z","formattedDate":"June 25, 2019","tags":[{"label":"autozi","permalink":"/blog/tags/autozi"},{"label":"zero-inflation","permalink":"/blog/tags/zero-inflation"}],"readingTime":21.945,"truncated":true,"authors":[{"name":"Oscar Clivio, Pierre Boyeau, Romain Lopez, Jeffrey Regier, Nir Yosef"}],"prevItem":{"title":"Hyperparameter search for scVI","permalink":"/blog/autotune"}}}]}')}}]);