(window.webpackJsonp=window.webpackJsonp||[]).push([[10],{81:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return l})),a.d(t,"metadata",(function(){return o})),a.d(t,"toc",(function(){return s})),a.d(t,"default",(function(){return b}));var i=a(3),n=a(7),r=(a(0),a(107)),l={},o={type:"mdx",permalink:"/press",source:"@site/src/pages/press.md"},s=[{value:"Articles",id:"articles",children:[]},{value:"Publications",id:"publications",children:[]},{value:"Awards",id:"awards",children:[]},{value:"Podcasts",id:"podcasts",children:[]},{value:"Talks",id:"talks",children:[]},{value:"Blog posts",id:"blog-posts",children:[]}],c={toc:s};function b(e){var t=e.components,a=Object(n.a)(e,["components"]);return Object(r.b)("wrapper",Object(i.a)({},c,a,{components:t,mdxType:"MDXLayout"}),Object(r.b)("h1",{id:"press"},"Press"),Object(r.b)("h2",{id:"articles"},"Articles"),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://www.nature.com/articles/s41592-018-0230-9&"}),Object(r.b)("em",{parentName:"a"},"Nature Methods"),": Bayesian deep learning for single-cell analysis"))),Object(r.b)("h2",{id:"publications"},"Publications"),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},"Joint probabilistic modeling of single-cell multi-omic data with totalVI.\nA. Gayoso","*",", Z. Steier","*",", R. Lopez, J. Regier, KL. Nazor, A. Streets\u2020, N Yosef\u2020.\n",Object(r.b)("em",{parentName:"li"},"Nature Methods (In press)"),", 2021"),Object(r.b)("li",{parentName:"ul"},"Probabilistic Harmonization and Annotation of Single-cell Transcriptomics data with Deep Generative Models. C. Xu","*",", R. Lopez","*",", E. Mehlman","*",", J. Regier, M.I. Jordan, N. Yosef.\n",Object(r.b)("em",{parentName:"li"},"Molecular Systems Biology"),", 2021"),Object(r.b)("li",{parentName:"ul"},"A joint model of unpaired data from scRNA-seq and spatial transcriptomics for imputing missing gene expression measurements.\nR. Lopez","*",", A. Nazaret","*",", M. Langevin","*",", J. Samaran, J. Regier, M.I. Jordan, N. Yosef.\n",Object(r.b)("em",{parentName:"li"},"ICML Workshop on Computational Biology"),", 2019"),Object(r.b)("li",{parentName:"ul"},"Deep Generative Models for Detecting Differential Expression in Single Cells.\nP. Boyeau, R. Lopez, J. Regier, A. Gayoso, MI. Jordan, N. Yosef.\n",Object(r.b)("em",{parentName:"li"},"Machine Learning in Computational Biology meeting"),", 2019"),Object(r.b)("li",{parentName:"ul"},"Detecting Zero-Inflated Genes in Single-Cell Transcriptomics Data.\nO. Clivio, R. Lopez, J. Regier, A. Gayoso, MI. Jordan, N Yosef.\n",Object(r.b)("em",{parentName:"li"},"Machine Learning in Computational Biology meeting"),", 2019"),Object(r.b)("li",{parentName:"ul"},"Deep Generative Modeling for Single-cell Transcriptomics.\nR. Lopez, J. Regier, MB. Cole, M. Jordan, N. Yosef.\n",Object(r.b)("em",{parentName:"li"},"Nature Methods"),", 2018")),Object(r.b)("h2",{id:"awards"},"Awards"),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://facultyopinions.com/prime/734522090"}),Object(r.b)("em",{parentName:"a"},"F1000 exceptional recommendation"),": Deep generative modeling for single-cell transcriptomics")),Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://sites.google.com/view/icml-compbio-2019/home"}),"ICML Workshop on Computational Biology, Best Poster Award, gimVI"))),Object(r.b)("h2",{id:"podcasts"},"Podcasts"),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://bioinformatics.chat/scvi"}),Object(r.b)("em",{parentName:"a"},"The Bioinformatics Chat Podcast"),": scVI with Romain Lopez and Gabriel Misrachi"))),Object(r.b)("h2",{id:"talks"},"Talks"),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://www.broadinstitute.org/talks/regev-lab-cell-circuits-and-epigenomics-and-mia-special-seminar-deep-generative-modeling"}),Object(r.b)("em",{parentName:"a"},"Regev Lab, Cell Circuits and Epigenomics, and MIA special seminar"),": Deep generative modeling for single-cell transcriptomics")),Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://ccbskillssem.github.io/pages/scanpy_scvi_tools/"}),Object(r.b)("em",{parentName:"a"},"Berkeley CCB skills seminar"),": Single-cell data analysis with Scanpy and scvi-tools"))),Object(r.b)("h2",{id:"blog-posts"},"Blog posts"),Object(r.b)("ul",null,Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://bair.berkeley.edu/blog/2018/12/05/genes/"}),Object(r.b)("em",{parentName:"a"},"BAIR Blog"),": Building Gene Expression Atlases with Deep Generative Models for Single-cell Transcriptomics")),Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://www.nxn.se/valent/2018/4/20/count-based-autoencoders-and-the-future-for-scrna-seq-analysis"}),Object(r.b)("em",{parentName:"a"},'What do you mean "heterogeneity"?'),": Count based autoencoders and the future for scRNA-seq analysis")),Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://www.nxn.se/valent/2020/9/16/a-trip-through-the-layers-of-scvi"}),Object(r.b)("em",{parentName:"a"},'What do you mean "heterogeneity"?'),": A trip through the layers of scVI")),Object(r.b)("li",{parentName:"ul"},Object(r.b)("a",Object(i.a)({parentName:"li"},{href:"https://saket-choudhary.me/blog/2020/12/27/scvi/"}),Object(r.b)("em",{parentName:"a"},"Saket Choudhary"),": Experiments with scVI"))))}b.isMDXComponent=!0}}]);