(window.webpackJsonp=window.webpackJsonp||[]).push([[10],{109:function(e,t,a){"use strict";a.d(t,"a",(function(){return p})),a.d(t,"b",(function(){return d}));var n=a(0),r=a.n(n);function i(e,t,a){return t in e?Object.defineProperty(e,t,{value:a,enumerable:!0,configurable:!0,writable:!0}):e[t]=a,e}function o(e,t){var a=Object.keys(e);if(Object.getOwnPropertySymbols){var n=Object.getOwnPropertySymbols(e);t&&(n=n.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),a.push.apply(a,n)}return a}function l(e){for(var t=1;t<arguments.length;t++){var a=null!=arguments[t]?arguments[t]:{};t%2?o(Object(a),!0).forEach((function(t){i(e,t,a[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(a)):o(Object(a)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(a,t))}))}return e}function c(e,t){if(null==e)return{};var a,n,r=function(e,t){if(null==e)return{};var a,n,r={},i=Object.keys(e);for(n=0;n<i.length;n++)a=i[n],t.indexOf(a)>=0||(r[a]=e[a]);return r}(e,t);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(n=0;n<i.length;n++)a=i[n],t.indexOf(a)>=0||Object.prototype.propertyIsEnumerable.call(e,a)&&(r[a]=e[a])}return r}var s=r.a.createContext({}),b=function(e){var t=r.a.useContext(s),a=t;return e&&(a="function"==typeof e?e(t):l(l({},t),e)),a},p=function(e){var t=b(e.components);return r.a.createElement(s.Provider,{value:t},e.children)},m={inlineCode:"code",wrapper:function(e){var t=e.children;return r.a.createElement(r.a.Fragment,{},t)}},u=r.a.forwardRef((function(e,t){var a=e.components,n=e.mdxType,i=e.originalType,o=e.parentName,s=c(e,["components","mdxType","originalType","parentName"]),p=b(a),u=n,d=p["".concat(o,".").concat(u)]||p[u]||m[u]||i;return a?r.a.createElement(d,l(l({ref:t},s),{},{components:a})):r.a.createElement(d,l({ref:t},s))}));function d(e,t){var a=arguments,n=t&&t.mdxType;if("string"==typeof e||n){var i=a.length,o=new Array(i);o[0]=u;var l={};for(var c in t)hasOwnProperty.call(t,c)&&(l[c]=t[c]);l.originalType=e,l.mdxType="string"==typeof e?e:n,o[1]=l;for(var s=2;s<i;s++)o[s]=a[s];return r.a.createElement.apply(null,o)}return r.a.createElement.apply(null,a)}u.displayName="MDXCreateElement"},80:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return o})),a.d(t,"metadata",(function(){return l})),a.d(t,"toc",(function(){return c})),a.d(t,"default",(function(){return b}));var n=a(3),r=a(7),i=(a(0),a(109)),o={},l={type:"mdx",permalink:"/press",source:"@site/src/pages/press.md"},c=[{value:"Articles",id:"articles",children:[]},{value:"Publications",id:"publications",children:[]},{value:"Awards",id:"awards",children:[]},{value:"Podcasts",id:"podcasts",children:[]},{value:"Talks",id:"talks",children:[]},{value:"Blog posts",id:"blog-posts",children:[]}],s={toc:c};function b(e){var t=e.components,a=Object(r.a)(e,["components"]);return Object(i.b)("wrapper",Object(n.a)({},s,a,{components:t,mdxType:"MDXLayout"}),Object(i.b)("h1",{id:"press"},"Press"),Object(i.b)("h2",{id:"articles"},"Articles"),Object(i.b)("ul",null,Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://www.nature.com/articles/s41592-018-0230-9&"}),Object(i.b)("em",{parentName:"a"},"Nature Methods"),": Bayesian deep learning for single-cell analysis"))),Object(i.b)("h2",{id:"publications"},"Publications"),Object(i.b)("ul",null,Object(i.b)("li",{parentName:"ul"},"Joint probabilistic modeling of single-cell multi-omic data with totalVI.\nA. Gayoso","*",", Z. Steier","*",", R. Lopez, J. Regier, KL. Nazor, A. Streets\u2020, N Yosef\u2020.\n",Object(i.b)("em",{parentName:"li"},"Nature Methods"),", 2021"),Object(i.b)("li",{parentName:"ul"},"Probabilistic Harmonization and Annotation of Single-cell Transcriptomics data with Deep Generative Models. C. Xu","*",", R. Lopez","*",", E. Mehlman","*",", J. Regier, M.I. Jordan, N. Yosef.\n",Object(i.b)("em",{parentName:"li"},"Molecular Systems Biology"),", 2021"),Object(i.b)("li",{parentName:"ul"},"A joint model of unpaired data from scRNA-seq and spatial transcriptomics for imputing missing gene expression measurements.\nR. Lopez","*",", A. Nazaret","*",", M. Langevin","*",", J. Samaran, J. Regier, M.I. Jordan, N. Yosef.\n",Object(i.b)("em",{parentName:"li"},"ICML Workshop on Computational Biology"),", 2019"),Object(i.b)("li",{parentName:"ul"},"Deep Generative Models for Detecting Differential Expression in Single Cells.\nP. Boyeau, R. Lopez, J. Regier, A. Gayoso, MI. Jordan, N. Yosef.\n",Object(i.b)("em",{parentName:"li"},"Machine Learning in Computational Biology meeting"),", 2019"),Object(i.b)("li",{parentName:"ul"},"Detecting Zero-Inflated Genes in Single-Cell Transcriptomics Data.\nO. Clivio, R. Lopez, J. Regier, A. Gayoso, MI. Jordan, N Yosef.\n",Object(i.b)("em",{parentName:"li"},"Machine Learning in Computational Biology meeting"),", 2019"),Object(i.b)("li",{parentName:"ul"},"Deep Generative Modeling for Single-cell Transcriptomics.\nR. Lopez, J. Regier, MB. Cole, M. Jordan, N. Yosef.\n",Object(i.b)("em",{parentName:"li"},"Nature Methods"),", 2018")),Object(i.b)("h2",{id:"awards"},"Awards"),Object(i.b)("ul",null,Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://facultyopinions.com/prime/734522090"}),Object(i.b)("em",{parentName:"a"},"F1000 exceptional recommendation"),": Deep generative modeling for single-cell transcriptomics")),Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://sites.google.com/view/icml-compbio-2019/home"}),"ICML Workshop on Computational Biology, Best Poster Award, gimVI"))),Object(i.b)("h2",{id:"podcasts"},"Podcasts"),Object(i.b)("ul",null,Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://bioinformatics.chat/scvi"}),Object(i.b)("em",{parentName:"a"},"The Bioinformatics Chat Podcast"),": scVI with Romain Lopez and Gabriel Misrachi"))),Object(i.b)("h2",{id:"talks"},"Talks"),Object(i.b)("ul",null,Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://www.broadinstitute.org/talks/regev-lab-cell-circuits-and-epigenomics-and-mia-special-seminar-deep-generative-modeling"}),Object(i.b)("em",{parentName:"a"},"Regev Lab, Cell Circuits and Epigenomics, and MIA special seminar"),": Deep generative modeling for single-cell transcriptomics")),Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://ccbskillssem.github.io/pages/scanpy_scvi_tools/"}),Object(i.b)("em",{parentName:"a"},"Berkeley CCB skills seminar"),": Single-cell data analysis with Scanpy and scvi-tools"))),Object(i.b)("h2",{id:"blog-posts"},"Blog posts"),Object(i.b)("ul",null,Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://bair.berkeley.edu/blog/2018/12/05/genes/"}),Object(i.b)("em",{parentName:"a"},"BAIR Blog"),": Building Gene Expression Atlases with Deep Generative Models for Single-cell Transcriptomics")),Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://www.nxn.se/valent/2018/4/20/count-based-autoencoders-and-the-future-for-scrna-seq-analysis"}),Object(i.b)("em",{parentName:"a"},'What do you mean "heterogeneity"?'),": Count based autoencoders and the future for scRNA-seq analysis")),Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://www.nxn.se/valent/2020/9/16/a-trip-through-the-layers-of-scvi"}),Object(i.b)("em",{parentName:"a"},'What do you mean "heterogeneity"?'),": A trip through the layers of scVI")),Object(i.b)("li",{parentName:"ul"},Object(i.b)("a",Object(n.a)({parentName:"li"},{href:"https://saket-choudhary.me/blog/2020/12/27/scvi/"}),Object(i.b)("em",{parentName:"a"},"Saket Choudhary"),": Experiments with scVI"))))}b.isMDXComponent=!0}}]);