(window.webpackJsonp=window.webpackJsonp||[]).push([[20],{109:function(e,t,a){"use strict";a.d(t,"a",(function(){return b})),a.d(t,"b",(function(){return m}));var r=a(0),n=a.n(r);function o(e,t,a){return t in e?Object.defineProperty(e,t,{value:a,enumerable:!0,configurable:!0,writable:!0}):e[t]=a,e}function i(e,t){var a=Object.keys(e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);t&&(r=r.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),a.push.apply(a,r)}return a}function c(e){for(var t=1;t<arguments.length;t++){var a=null!=arguments[t]?arguments[t]:{};t%2?i(Object(a),!0).forEach((function(t){o(e,t,a[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(a)):i(Object(a)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(a,t))}))}return e}function l(e,t){if(null==e)return{};var a,r,n=function(e,t){if(null==e)return{};var a,r,n={},o=Object.keys(e);for(r=0;r<o.length;r++)a=o[r],t.indexOf(a)>=0||(n[a]=e[a]);return n}(e,t);if(Object.getOwnPropertySymbols){var o=Object.getOwnPropertySymbols(e);for(r=0;r<o.length;r++)a=o[r],t.indexOf(a)>=0||Object.prototype.propertyIsEnumerable.call(e,a)&&(n[a]=e[a])}return n}var s=n.a.createContext({}),p=function(e){var t=n.a.useContext(s),a=t;return e&&(a="function"==typeof e?e(t):c(c({},t),e)),a},b=function(e){var t=p(e.components);return n.a.createElement(s.Provider,{value:t},e.children)},u={inlineCode:"code",wrapper:function(e){var t=e.children;return n.a.createElement(n.a.Fragment,{},t)}},d=n.a.forwardRef((function(e,t){var a=e.components,r=e.mdxType,o=e.originalType,i=e.parentName,s=l(e,["components","mdxType","originalType","parentName"]),b=p(a),d=r,m=b["".concat(i,".").concat(d)]||b[d]||u[d]||o;return a?n.a.createElement(m,c(c({ref:t},s),{},{components:a})):n.a.createElement(m,c({ref:t},s))}));function m(e,t){var a=arguments,r=t&&t.mdxType;if("string"==typeof e||r){var o=a.length,i=new Array(o);i[0]=d;var c={};for(var l in t)hasOwnProperty.call(t,l)&&(c[l]=t[l]);c.originalType=e,c.mdxType="string"==typeof e?e:r,i[1]=c;for(var s=2;s<o;s++)i[s]=a[s];return n.a.createElement.apply(null,i)}return n.a.createElement.apply(null,a)}d.displayName="MDXCreateElement"},90:function(e,t,a){"use strict";a.r(t),a.d(t,"frontMatter",(function(){return i})),a.d(t,"metadata",(function(){return c})),a.d(t,"toc",(function(){return l})),a.d(t,"default",(function(){return p}));var r=a(3),n=a(7),o=(a(0),a(109)),i={},c={type:"mdx",permalink:"/get_started",source:"@site/src/pages/get_started.md"},l=[{value:"Basic installation",id:"basic-installation",children:[]},{value:"Single-cell omics data analysis",id:"single-cell-omics-data-analysis",children:[{value:"scRNA-seq",id:"scrna-seq",children:[]},{value:"CITE-seq",id:"cite-seq",children:[]},{value:"Spatial transcriptomics",id:"spatial-transcriptomics",children:[]},{value:"scATAC-seq",id:"scatac-seq",children:[]}]},{value:"Methods developer toolkit",id:"methods-developer-toolkit",children:[]},{value:"Resources",id:"resources",children:[]}],s={toc:l};function p(e){var t=e.components,a=Object(n.a)(e,["components"]);return Object(o.b)("wrapper",Object(r.a)({},s,a,{components:t,mdxType:"MDXLayout"}),Object(o.b)("h1",{id:"get-started"},"Get Started"),Object(o.b)("p",null,Object(o.b)("inlineCode",{parentName:"p"},"scvi-tools")," (single-cell variational inference tools) is a package for probabilistic modeling of single-cell omics data, built on top of ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://pytorch.org/"}),"PyTorch")," and ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://anndata.readthedocs.io/en/latest/"}),"AnnData"),". The package hosts implementations of several models that perform a wide range of single-cell data analysis tasks, as well as the building blocks to rapidly prototype new probabilistic models."),Object(o.b)("h2",{id:"basic-installation"},"Basic installation"),Object(o.b)("p",null,Object(o.b)("inlineCode",{parentName:"p"},"scvi-tools")," can be installed with ",Object(o.b)("inlineCode",{parentName:"p"},"pip")," or ",Object(o.b)("inlineCode",{parentName:"p"},"conda"),":"),Object(o.b)("pre",null,Object(o.b)("code",Object(r.a)({parentName:"pre"},{}),"pip install scvi-tools\n")),Object(o.b)("pre",null,Object(o.b)("code",Object(r.a)({parentName:"pre"},{}),"conda install scvi-tools -c bioconda -c conda-forge\n")),Object(o.b)("p",null,"To leverage a GPU for inference, a version of ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://pytorch.org/get-started/locally/"}),"PyTorch")," that supports your GPU will need to be installed separately."),Object(o.b)("h2",{id:"single-cell-omics-data-analysis"},"Single-cell omics data analysis"),Object(o.b)("p",null,Object(o.b)("inlineCode",{parentName:"p"},"scvi-tools")," hosts implementations of many models, with a convenient interface to ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://scanpy.readthedocs.io/en/stable/"}),"scanpy"),", ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://satijalab.org/seurat/"}),"Seurat")," and ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://www.bioconductor.org/"}),"Bioconductor")," (refer to our tutorials to learn about our R support):"),Object(o.b)("h3",{id:"scrna-seq"},"scRNA-seq"),Object(o.b)("ul",null,Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://rdcu.be/bdHYQ"}),"scVI")," for analysis of single-cell RNA-seq data, as well as its improved ",Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://www.biorxiv.org/content/biorxiv/early/2019/10/04/794289.full.pdf"}),"differential expression framework"),"."),Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://academic.oup.com/bioinformatics/article/36/11/3418/5807606"}),"LDVAE")," for an interpretable linear factor model version of scVI."),Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://www.embopress.org/doi/full/10.15252/msb.20209620"}),"scANVI")," for cell annotation of scRNA-seq data using semi-labeled examples."),Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://www.nature.com/articles/s41592-019-0529-1"}),"CellAssign")," for cell annotation of scRNA-seq data using marker genes. "),Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://www.sciencedirect.com/science/article/pii/S2405471220301952"}),"SOLO")," for detection of doublets."),Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://www.biorxiv.org/content/10.1101/2020.07.16.205997v1"}),"scArches")," for transfer learning when learning cell embeddings in an online fashion (implemented for scVI, scANVI & totalVI)")),Object(o.b)("h3",{id:"cite-seq"},"CITE-seq"),Object(o.b)("ul",null,Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://www.nature.com/articles/s41592-020-01050-x.epdf?sharing_token=F_xpBUwgD5vHVAB5jvU0c9RgN0jAjWel9jnR3ZoTv0NDA6nPyssLQkWDcY8VB7gLKdlD8q3cqpF5ovLZjZTudaweTnzODEXCrkjJ2UJEEpOAhfOCwEZ0h0Y2WTkn_GOBfGUZhH-lzX_u_AswArGFGinU1pcuXPvuTkPrrQI4zko%3D"}),"totalVI")," for analysis of CITE-seq data.")),Object(o.b)("h3",{id:"spatial-transcriptomics"},"Spatial transcriptomics"),Object(o.b)("ul",null,Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://www.nature.com/articles/s42003-020-01247-y"}),"Steroscope")," for deconvolution of spatial transcriptomics data."),Object(o.b)("li",{parentName:"ul"},Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://arxiv.org/pdf/1905.02269.pdf"}),"gimVI")," for imputation of missing genes in spatial transcriptomics from scRNA-seq data.")),Object(o.b)("h3",{id:"scatac-seq"},"scATAC-seq"),Object(o.b)("ul",null,Object(o.b)("li",{parentName:"ul"},"peakVI for analysis of single-cell ATAC sequencing profiles")),Object(o.b)("h2",{id:"methods-developer-toolkit"},"Methods developer toolkit"),Object(o.b)("p",null,Object(o.b)("inlineCode",{parentName:"p"},"scvi-tools")," is a high-level probabilistic programming package especially developped for single-cell omics data. We hope it will empower method developers and help them focus on building fruitful models. To achieve this, we host black-box primitives for loading data from ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://anndata.readthedocs.io/en/latest/"}),"AnnData"),", black-box training procedures using ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://www.pytorchlightning.ai/"}),"PyTorch Lightning"),", and optionally provides automatic inference recipes based on ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://pyro.ai/"}),"Pyro"),". We currently support many optimization-based inference procedures including SVI, AEVB, MAP, MLE and EM. To learn more about the developer toolkit, please visit our ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://docs.scvi-tools.org/en/latest/user_guide/index.html"}),"tutorials"),". We also provide an example ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://github.com/YosefLab/scvi-tools-skeleton/"}),"skeleton")," implementation of a basic scVI in PyTorch and Pyro as an independant package based off ",Object(o.b)("inlineCode",{parentName:"p"},"scvi-tools"),". If you wish to add an implementation of your algorithm in scvi-tools, please post a ",Object(o.b)("a",Object(r.a)({parentName:"p"},{href:"https://github.com/YosefLab/scvi-tools/issues"}),"GitHub issue")," to reach out to us."),Object(o.b)("h2",{id:"resources"},"Resources"),Object(o.b)("ul",null,Object(o.b)("li",{parentName:"ul"},"Tutorials, API reference, and advanced installation guides are available in the ",Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://docs.scvi-tools.org/en/stable/"}),"docs"),"."),Object(o.b)("li",{parentName:"ul"},"For discussion of usage, checkout out our ",Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://discourse.scvi-tools.org/"}),"forum"),"."),Object(o.b)("li",{parentName:"ul"},"For immediate public chat with the developers, come talk to us on ",Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://gitter.im/scvi-tools/"}),"Gitter"),"."),Object(o.b)("li",{parentName:"ul"},"For a skeleton of implementation of a new model in scvi-tools, refer to our ",Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://github.com/YosefLab/scvi-tools-skeleton/"}),"template repository")),Object(o.b)("li",{parentName:"ul"},"To report bugs, make an issue on ",Object(o.b)("a",Object(r.a)({parentName:"li"},{href:"https://github.com/YosefLab/scvi-tools/issues"}),"GitHub"),".")))}p.isMDXComponent=!0}}]);