# Press

## General

- [*Nature Methods*: Bayesian deep learning for single-cell analysis](https://www.nature.com/articles/s41592-018-0230-9&)


## Awards

- [*F1000 exceptional recommendation*: Deep generative modeling for single-cell transcriptomics](https://facultyopinions.com/prime/734522090)
- [ICML Workshop on Computational Biology, Best Poster Award, gimVI](https://sites.google.com/view/icml-compbio-2019/home)
## Podcasts

- [*The Bioinformatics Chat Podcast*: scVI with Romain Lopez and Gabriel Misrachi](https://bioinformatics.chat/scvi)

## Talks

- [*Regev Lab, Cell Circuits and Epigenomics, and MIA special seminar*: Deep generative modeling for single-cell transcriptomics](https://www.broadinstitute.org/talks/regev-lab-cell-circuits-and-epigenomics-and-mia-special-seminar-deep-generative-modeling)
- [*Berkeley CCB skills seminar*: Single-cell data analysis with Scanpy and scvi-tools](https://ccbskillssem.github.io/pages/scanpy_scvi_tools/)

## Blog posts

- [*BAIR Blog*: Building Gene Expression Atlases with Deep Generative Models for Single-cell Transcriptomics](https://bair.berkeley.edu/blog/2018/12/05/genes/)
- [*What do you mean "heterogeneity"?*: Count based autoencoders and the future for scRNA-seq analysis](https://www.nxn.se/valent/2018/4/20/count-based-autoencoders-and-the-future-for-scrna-seq-analysis)
- [*What do you mean "heterogeneity"?*: A trip through the layers of scVI](https://www.nxn.se/valent/2020/9/16/a-trip-through-the-layers-of-scvi)
- [*Saket Choudhary*: Experiments with scVI](https://saket-choudhary.me/blog/2020/12/27/scvi/)