---
slug: v090
title:      scvi-tools 0.9.0 release
date:       2021-03-03
author: Adam Gayoso, Romain Lopez, Galen Xing, Nir Yosef
tags: [scvi-tools, release]
---
import useBaseUrl from '@docusaurus/useBaseUrl';

Today we officially released `scvi-tools` version 0.9.0 ([changelog](https://docs.scvi-tools.org/en/stable/release_notes/index.html)). This release marks the culmination of five months of work on the backend of the codebase, which came after three months of work on the frontend.
In this short note, we officially introduce `scvi-tools` as a readily usable codebase that contains many implementations of probabilistic single-cell omics methods, and also features a high-level interface to accelerate the model development process. We start with some historical notes about our previous codebase, which was mostly used for internal developments in the last three years. We then describe the obstacles we found to its external adoption, and the foundational idea behind the new `scvi-tools` work: a high-level deep probabilistic programming library specialized for single-cell omics data.

<!--truncate-->

## Taking a step back: the original `scvi` codebase

Many members of the Yosef Lab, and in particular Jeff Regier, Edouard Melhman, Romain, and Adam helped conceive, develop, and maintain the `scvi` codebase. The initial philosophy was to make the code available for users to run [scVI](https://www.nature.com/articles/s41592-018-0229-2), but also to have a proper codebase for developing novel algorithms for single-cell omics analysis (e.g., [totalVI](https://www.nature.com/articles/s41592-020-01050-x) was developed this way). Over the last three years, we have hosted seven visiting graduate students who wrote their master theses in the Yosef Lab by building new functionalities, as well as new algorithms (including [scANVI](https://www.embopress.org/doi/full/10.15252/msb.20209620), [AutoZI](https://www.biorxiv.org/content/biorxiv/early/2019/10/10/794875.full.pdf) and [gimVI](https://arxiv.org/pdf/1905.02269.pdf)) directly into the `scvi` codebase. However, our ambition at the time was to solely use this infrastructure for internal developments. Consequently, some mistakes were made, in which we did not make use of the existing single-cell analysis infrastructure. For example, we had built manual data formatting boilerplate code to read many different single-cell omics input formats.

## Identifying key improvements, creation of `scvi-tools`

Even though we were actively branding the `scvi` codebase as a framework for creating new probabilistic models, we did not encounter clear success in this area, except for the `LDVAE` model which was our first model developed by an external user of the codebase (thanks [Valentine](https://www.nxn.se/)!). Also, we became aware of some suboptimal API choices we had made, and how we could work on the user's perspective in order to make the codebase more attractive. At this point in time, Galen joined the Yosef lab and we rethought the foundations of `scvi-tools`:

1. Host reimplementations of existing methods that are not currently easy to use.
2. For all methods, provide a consistent and simplified user experience, and provide vignettes.
3. Focus on [`AnnData`](https://anndata.readthedocs.io/en/latest/) for the input data format, and use [`Scanpy`](https://scanpy.readthedocs.io/en/stable/) for all other processing.
4. Build explicit tutorials to show interaction with the R ecosystem ([`Bioconductor`](https://www.bioconductor.org/), [`Seurat`](https://satijalab.org/seurat/)).
5. Rewrite all the the training code, relying instead on [PyTorch Lightning](https://www.pytorchlightning.ai/).
6. Add an interface to [Pyro](https://pyro.ai/), in order to further automate inference.
7. Build [tutorials](https://docs.scvi-tools.org/en/latest/user_guide/index.html) for model developers.

Today, most of those features are already usable. Visit our [landing page](https://scvi-tools.org)! We now provide details for some of those developments, present in `scvi-tools` v0.9.0.

## Focus on the model

In this release, we introduced a new paradigm for building single-cell focused probabilistic models in which model development is hyper-focused on the model at hand. Based on our experience of building variational autoencoders for single-cell data, we identified several opportunties to abstract boilerplate code in a reusable way. Therefore, we built objects in the scvi-tools codebase to handle auxiliary tasks such as data loading, training, save/load, and device management. As an example, we wrote the `scvi.data` module to handle `AnnData` objects, including registration of model-specific tensors and generic "ann"-data loading into models. Consequently, model development solely focuses on (1) defining a probabilstic model/inference scheme and (2) expressing it in a structured way based on our abstract classes.

## New implementations

With these new model building blocks, we were able to implement models external to our lab with relative ease. This included [Stereoscope](https://www.nature.com/articles/s42003-020-01247-y) for deconvolving spatial transcriptomics data, [Solo](https://www.sciencedirect.com/science/article/pii/S2405471220301952) for detection of doublets in scRNA-seq data, and [CellAssign](https://www.nature.com/articles/s41592-019-0529-1) for reference-based annotation of scRNA-seq data. These implementations required significantly fewer lines of code with `scvi-tools`.

## Pyro integration

Another feature we are excited about is integration with [Pyro](https://pyro.ai/), which further abstracts the process of manually deriving optimization objectives, thus allowing for complex hierarchical Bayesian models. Relying on Pyro to power a model is completely optional, but we feel there will be situations where it is clearly advantageous to use it.

## Template repository

We also wrote a [template](https://scvi-tools-skeleton.readthedocs.io/en/stable/) GitHub repository to accelerate the package creation process. This includes template code to setup documentation, continuous integration testing, and popular code styling practices.

## New longitudinal features

Finally, by having implementations of all these models in the codebase, we were able to broadcast new features across models. This included support for multiple (continuous or categorical) covariates when integrating data with scVI, scANVI, or totalVI. We are excited to see the impact of non-linear dataset integration extended in this way, and have already seen promising results in correcting, e.g., cell cycle effects.
We also have extended the [scArches](https://scarches.readthedocs.io/en/latest/) method for query/reference dataset integration to the scVI, scANVI, and totalVI models. This required implementing one [`Mixin`](https://github.com/YosefLab/scvi-tools/blob/master/scvi/model/base/_archesmixin.py) class with the core transfer learning logic.
